### PR TITLE
E6V2 BMC OE

### PR DESCRIPTION
**Pull request by mechmerlin**
## Description

File renames and documentation changes. No code change. 

Apparently Exclusive sold a different BMC pcb for Round 2. What we have here in QMK is actually the R1 PCB. Renamed and minor documentation changes to specify the differences

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/exclusive/e6v2/readme.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The E6-V2 is a 60% keyboard manufactured by Exclusive.
 
 These docs are for the QMK version of the E6-V2 PCB. [More info on qmk.fm](http://qmk.fm/)
```
Comment (noroadsleft) [inline keyboards/exclusive/e6v2/readme.md]: I find this line a bit misleading, considering the BMC PCBs can still run QMK.

File: keyboards/exclusive/e6v2/readme.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ The E6-V2 is a 60% keyboard manufactured by Exclusive.
...
 
-The E6V2 has been available with either a bootmapper client or QMK powered PCB. During the second round, the QMK powered PCB was redesigned and used different ports and a different switch matrix. 
+The E6V2 has been available with either a bootmapper client or QMK powered PCB. During the second round, the QMK powered and BMC PCB were redesigned and used different ports and a different switch matrix.
```
Comment (noroadsleft) [inline keyboards/exclusive/e6v2/readme.md]: Tiny bit of wordsmithing.

```suggestion
The E6V2 has been available with either a Bootmapper Client or QMK-powered PCB. During the second round, the QMK-powered and BMC PCBs were redesigned, and used different ports and a different switch matrix. 
```

File: keyboards/exclusive/e6v2/readme.md (hunk 1 above)
Comment (mechmerlin) [reply keyboards/exclusive/e6v2/readme.md]: It is misleading but that is actually what they are called in the GB. I'm trying to keep the wording consistent with what the GB says and what Exclusive has communicated to me.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**